packages/fcl-config/README.md

```diff
@@ -0,0 +1,16 @@
+# @onflow/fcl-config
+
+Recursively searches synchronously for a `flow.json` file, starting in the execution context moving towards the home directory inclusively.
+
+# Install
+
+```bash
+npm install --save @onflow/fcl-config
+```
+
+# Usage
+
+```javascript
+import {flowConfig} from "@onflow/fcl-config"
+const config = flowConfig() // javascript object if found, error if not found
```

Comment:
Hmm, maybe this could be named `findFlowConfig()` to be a bit more descriptive about what this is doing?

Reply:
its actually not always finding it, its returning a memoized copy of it if it already has it, searches for it if it doesn't have it and then memoizes it or it throws an error if it cant find it. I believe this to be an accurate name.